utility/tautulli_friendly_name_to_ombi_alias_sync.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""
+Description: Sync Tautulli friendly names with Ombi aliases (Tautulli as master)
+Author: DirtyCajunRice
+Requires: requests, python3.6+
+"""
+from requests import Session
+from plexapi.server import CONFIG
+from urllib3 import disable_warnings
+from urllib3.exceptions import InsecureRequestWarning
+
+OMBI_BASEURL = ''
+OMBI_APIKEY = ''
+
+TAUTULLI_BASEURL = ''
+TAUTULLI_APIKEY = ''
+
+# Dont Edit Below #
+TAUTULLI_BASEURL = TAUTULLI_BASEURL or CONFIG.data['auth'].get('tautulli_baseurl')
+TAUTULLI_APIKEY = TAUTULLI_APIKEY or CONFIG.data['auth'].get('tautulli_apikey')
+OMBI_BASEURL = OMBI_BASEURL or CONFIG.data['auth'].get('ombi_baseurl')
+OMBI_APIKEY = OMBI_APIKEY or CONFIG.data['auth'].get('ombi_apikey')
+
+disable_warnings(InsecureRequestWarning)
+SESSION = Session()
+SESSION.verify = False
+
+HEADERS = {'apiKey': OMBI_APIKEY}
+PARAMS = {'apikey': TAUTULLI_APIKEY, 'cmd': 'get_users'}
+
+TAUTULLI_USERS = SESSION.get('{}/api/v2'.format(TAUTULLI_BASEURL.rstrip('/')), params=PARAMS).json()['response']['data']
```

Comment:
Wouldn't you want to use f-string instead of format since your using py3? 
```python
TAUTULLI_USERS = SESSION.get(f'{TAUTULLI_BASEURL.rstrip('/')}/api/v2', params=PARAMS).json()['response']['data']
```
or
```python
TAUTULLI_BASEURL = TAUTULLI_BASEURL.rstrip('/')
TAUTULLI_USERS = SESSION.get(f'{TAUTULLI_BASEURL}/api/v2', params=PARAMS).json()['response']['data']
```

Reply:
ive been trying not to use fstrings in these scripts because lots of repos only have 3.5.3 which doesnt have fstrings yet. making sure its compatible with all py3. (had enough issues helping people install 3.6.2+ in varken)